### PR TITLE
Stack and queue

### PR DESCRIPTION
**Pull request by tcoots88**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401challenges/src/test/java/code401challenges/stacksandqueues/QueueTest.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        assertTrue("peek should return a value of 1", testQueue.peek() == 1);
+        System.out.println("Front value is " + testQueue.front.value);
+        System.out.println("Party on, Garth");
+    }
+
+    @Test
+    public void testIsEmptyOnEmptyQueue(){
+        Queue testQueue = new Queue();
+        assertEquals("isEmpty should return true for this empty queue",true, testQueue.isEmpty());
+        System.out.println("Front value is " + testQueue.front.value);
```
Comment (JamesDansie) [inline code401challenges/src/test/java/code401challenges/stacksandqueues/QueueTest.java]: Replace testQueue.front.value with testQuest.front, otherwise you'll get a null pointer

File: code401challenges/src/main/java/code401challenges/stacksandqueues/Stack.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+public class Stack {
+    Node top;
+
+    public Stack (){
+        top = null;
+    }
+
+    public void push (int value){
+        Node newNode = new Node(value);
```
Comment (JamesDansie) [inline code401challenges/src/main/java/code401challenges/stacksandqueues/Stack.java]: this can be shorter;
this.top = new Node(value, this.top);
